client/src/pages/Home/MyEventsView/index.tsx

```diff
@@ -185,9 +206,91 @@ export default function MyEventsView() {
         overflow: 'hidden',
         justifyContent: 'center',
         flexDirection: 'column',
+        px: 2,
       }}
     >
-      {events.length === 0 ? (
+      {/* Date Navigator */}
```

Comment:
better to create a new component in the same file called DateNavigator that takes a children as prop. that would help make this component smaller